assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -86,18 +81,14 @@ export default function SettingsForm() {
 				</div>
 			) }
 
-			{ ( isGA4Enabled && ( SETUP_FLOW_MODE_GA4_TRANSITIONAL === setupFlowMode || SETUP_FLOW_MODE_GA4 === setupFlowMode ) ) && (
-				<GA4PropertyNotice notice={ __( 'You’ll need to connect the Google Analytics 4 property that’s associated with this Universal Analytics property.', 'google-site-kit' ) }>
+			{ ( isGA4Enabled && !! ga4PropertyID ) && (
+				<GA4PropertyNotice notice={ __( 'You need to connect the Google Analytics 4 property that’s associated with this Universal Analytics property.', 'google-site-kit' ) }>
 					<div className="googlesitekit-setup-module__inputs">
 						<GA4PropertySelect />
 					</div>
 				</GA4PropertyNotice>
 			) }
 
-			{ ( isGA4Enabled && SETUP_FLOW_MODE_UA === setupFlowMode ) && (
-				<GA4PropertyNotice notice={ __( 'A Google Analytics 4 property will also be created.', 'google-site-kit' ) } />
-			) }
```

Comment:
Should this be removed?

Reply:
I think you are right, this shouldn't be removed. Reverted it.